english/invite-friends/flip/flip.md

```diff
@@ -0,0 +1,5 @@
+You attempt a flip but only manage to screw up and land on your head. Despite being in what is essentially a bounce house made of soft marshmallows, the stunt ends up breaking your neck. 
+
+You lay paralyzed on the floor, unable to feel the fluffiness of the surface that so recently betrayed you, and your friends leave, bummed that you managed to ruin another party. 
```

Comment:
@impossibleshapes, notice how the line numbers are 1, 2, and 3, where 2 is blank? Line 2 is actually unnecessary. (Before you made the split, all the content was on line 1, so breaking this up into line 1 and line 2 would be sufficient without adding a blank line. If you don't have line numbers in your editor, you might want to Google for how to end them so you can see the difference locally.)

If you don't add blank lines, the display on GitHub won't be any different, and it's only the plain text file that looks different. This means you don't have to search for a natural breaking point to end lines, but can just add one whenever a line starts to get too long. Hopefully this should make it easier to keep to the 120 character maximum.